UITests/Receipts/ListAllReceiptsTests.swift

```diff
@@ -185,13 +187,14 @@ class ListAllReceiptsTests: BaseTests {
                              filename: "ReceiptsForOneMonth",
                              method: HTTPMethod.get)
 
-        mockServer.setupStub(url: ppcReceiptsURL,
+        mockServer.setupStub(url: "/rest/v3/users/usr-token/prepaid-cards/trm-token2/receipts",
```

Comment:
Kindly update `ppcReceiptsURL` to `ppcPrimaryReceiptsURL` and define the new url and use it everywhere in the test

Reply:
sure, i will update.